routing_model/index.html

```diff
@@ -653,7 +653,8 @@ <h2>Relocating a member of our section away from it</h2>
   </p>
   <button class="collapsible">get_best_relocating_node_and_target</button>
   <div class="content">
-    <p>Returns the best node to relocate and the target address to send it to<br/>
+    <p>Takes ALREADY_RELOCATING for the nodes it will ignore.<br/>
```

Comment:
As it is stated, I think it's a bit unclear. If I understand the general idea right, if ALREADY_RELOCATING is not empty, this function will always return None to reduce the number of occasions where more than one nodes are relocating away from our section.

This is not how that sentence reads for me.

Reply:
No, if ALREADY_RELOCATING is not empty, that function will not return a node that is in ALREADY_RELOCATING, it will return the next node to relocate that is not in ALREADY_RELOCATING.